waggle-dance-integration-tests/src/test/java/com/hotels/bdp/waggledance/metrics/MonitoringConfigurationTest.java

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
Ah OK, so we do still have a MonitoringConfigurationTest. Why does GH show this as a delete and new rather than a change? Did something move?

Reply:
move from wd-core, wd-integration-tests. probably should be back in core

Reply:
MonitoringConfigurationIntegrationTest which was here is now deleted as MonirotingConfiguration is not doing anything. I have moved this to wd-core.